Tutorials/Creating-Basic-Site/Articles-Parent-and-Article-Items/index.md

```diff
@@ -2,109 +2,132 @@
+3. Click **Document Type with Template**.
+    ![Document type with template](images/Document_type_with_template.png)
+4. Enter a **Name** for the **Document Type**. Let's call it _Articles Item_.
+5. Let's add two fields with the following specifications:
+    | Group   | Field Name      | Alias          | Data Type        |
+    |---------|-----------------|----------------|------------------|
+    | Content | Article Title   | articleTitle   | TextBox          |
+    | Content | Article Content | articleContent | Rich Text Editor |
+
+    ![Article Item Document Type Data Properties](images/figure-39-articles-item-v8.png){: data-fancybox}
+6.  Navigate to the **Home Page** node and go to the **Permissions** tab.
+7. In the **Allowed child node types**, click **Add child**. The **Choose child node** window opens.
+8. Select **Articles Main** and click **Save**.
+
+## Creating the Content Node
+
+To add a content node:
+
+1. Go to **Content**.
+2. Click the **...** next to the **HomePage** and select **Articles Main**.
+3. Enter the name for the article. We are going to call it _Articles_.
+4. Enter the **Article Title**, **Article Content**, and click **Save**.
+   When you click on Save, you will notice a list view is created. The child nodes are placed in a list making it easier to view them. You can create new nodes from this section.
+   
+   :::tip
+    If you do not see the list view, try refreshing the page.
+   :::
+5. Let's add two child nodes called **Article 1**, **Article 2**, and click **Save**.
+    ![Content Tree With Articles](images/figure-40-articles-created-v8.png)
+
+## Updating the Template
+
+To update the **Articles Main** template, follow these steps:
+
+1. Go to **Settings**.
+2. In the **Templating** section, expand the **Templates** folder. You should see a template titled **_Articles Main_**.
+3. In the **Master template** select **Master** and click **Save**.
+4. Copy the template content from the **_Simple Content Page_**  template and paste the content into **Articles Main**.
+    
+    :::warn
+    Take care when pasting the template not to overwrite the first line `@inherits Umbraco.Web.Mvc.UmbracoViewPage<ContentModels.ArticlesMain>`. If you get an error when loading the page ensure the last part in <> brackets matches your Document Type alias.
+    :::
+5. Replace the static text within the `<!-- Jumbotron, w title -->` tags with the Model.Value references. For example:  Add references to **_articlesTitle_** and **_articlesBodyText_**.
+ ![Articles Main Template](images/articles-main-template.png)
+6. Define a query for all articles, just below the `<div>` tag of the`<!-- Jumbotron, w title -->`.
+    ![Query Builder](images/query-builder.png)
+7. You can set conditions to get specific articles or decide the order of the articles. For the purpose of this guide, we'll use the following parameters:
+    ![Query parameters](images/query-parameters.png)
```

Comment:
This image shows that there should be 3 articles in the query, but the guide only says to create two 🤔 
Make sure all images/screenshots are consistent with the text content!